utopia-remix/app/routes/internal.editor.projects.$id.sharing.tsx

```diff
@@ -73,17 +88,50 @@ const EditorProjectSharingPage = React.memo(() => {
   )
 
   return (
-    <ProjectsContext.Provider value={store}>
-      <SharingDialogContent
-        project={data.project}
-        accessRequestsState={accessRequests.state}
-        accessLevel={accessLevel}
-        changeProjectAccessLevel={changeProjectAccessLevel}
-        asDialog={false}
-      />
-    </ProjectsContext.Provider>
+    <SharingDialogContent
+      project={data.project}
+      accessRequestsState='ready'
+      accessLevel={accessLevel}
+      changeProjectAccessLevel={changeProjectAccessLevel}
+      asDialog={false}
+    />
   )
 })
 EditorProjectSharingPage.displayName = 'EditorProjectSharingPage'
 
-export default EditorProjectSharingPage
+// Revalidate the loader at fixed intervals of the given timeout milliseconds.
+function useRevalidateWithInterval(timeout: number) {
```

Comment:
good solution. feels like we can maybe have a generic hook here (reavalidate loader => check if changed => callback) but it might be over-engineering now :)